engines/director/detection_tables.h

```diff
@@ -2400,7 +2401,7 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACGAME1("cinevolt", "", "Cinema Volta", "2ce360c9ea2da80a2c6d1040b0ad92dd", 387911, 313),
 	WINGAME1("cinevolt", "", "CINEVOLT.EXE", "65d06b5fef155a2473434571aff5bc29", 370009, 313),
 
-	WINGAME1_l("cklasse", "", "c-klasse.exe", "39df1c7ff23ba627ddffaa5a77f83008", 376244, Common::DE_DEU, 313),
+	WINGAME1_l("cklasse", "v1.0", "c-klasse.exe", "39df1c7ff23ba627ddffaa5a77f83008", 376244, Common::DE_DEU, 313),
```

Comment:
Were ther other versions released? If not, then it does not make sense to make this difference.

Reply:
This is the Demo from the ScummVM Director Demo site, when you exit the "Game" it says Version 1.0 from 1993, so i am not sure if its the only version that exists.

Reply:
Well, if it was the only one version release, there is no sense in adding this to the extra